modules/primer-marketing-utilities/docs/layout.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Layout
+status: New release
+status_issue: https://github.com/github/design-systems/issues/325
+---
+
+The following layout utilities are meant to used in addition to those within primer-core.
+
+{:toc}
+
+## Responsive position
+
+These position utilities behave the same way as the positioning utilities in primer-core, however they include responsive states so that they can be used at specific viewport widths.
```

Comment:
It's good to start with how to use the style before getting into details such as referencing how it works with other styles. I basically try and write docs with the assumption that people will put up with reading information for about 5 seconds 🙃. Suggest something like:

> Use responsive position utilities to adjust the position of an element at different breakpoints. These can be used in addition to the [position utilities]((layout)) available in primer-core.